new-dti-website/components/apply/ApplyFAQ.tsx

```diff
@@ -26,6 +26,7 @@ const ApplyFAQ = () => {
           section === question ? 'text-[#FEFEFE] bg-[#0C0404]' : ''
         }`}
       onClick={() => setQuestion(section)}
+      aria-label={`select ${section} section`}
```

Comment:
yay for aria labels!

Just a small nit i found when testing – do you think we could make the `<button>` wrap the label as well ("Design", "Business", etc.)? That way you can click on the entire container (label + icon).

It's just a smol polish thing that could increase the tap target and make it easier to select the tabs


![Screenshot 2024-11-30 at 12 41 14 AM](https://github.com/user-attachments/assets/30d980f6-d1aa-4e79-bcc1-c917ef1c855c)